bat/lib/mixerlib.bash

```diff
@@ -99,9 +99,7 @@ download-rpm() {
 get-last-format-boundary() {
   latest=$(curl https://download.clearlinux.org/latest)
   format=$(curl https://download.clearlinux.org/update/$latest/format)
-  first=$(curl https://download.clearlinux.org/update/version/format$format/first)
-
-  echo $(($first-10))
+  echo $(curl https://download.clearlinux.org/update/version/format$(($format-1))/latest)
```

Comment:
I guess this is ok since it is getting it directly from upstream. If format was a default `1` this wouldn't work of course.